### PR TITLE
Fix dynamo benchmark skip logic for cpu device

### LINKED ISSUE #132380: [inductor][cpu] LayoutLMForSequenceClassification amp single thread accuracy failure
### 🐛 Describe the bug

[inductor_single_run.sh](https://github.com/chuanqi129/inductor-tools/blob/main/scripts/modelbench/inductor_single_run.sh)

The Bad Commit:
8458980bbf78714a0fbe703785c100cad523fade
```
/workspace/pytorch# bash inductor_single_run.sh single inference accuracy huggingface LayoutLMForSequenceClassification amp
Testing with freezing on.
single-thread testing....
/opt/conda/lib/python3.8/site-packages/huggingface_hub/file_download.py:1150: FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
  warnings.warn(
loading model: 0it [00:01, ?it/s]
cpu  eval  LayoutLMForSequenceClassification
WARNING:common:Trying to call the empty_gpu_cache for device: cpu, which is not in list [cuda, xpu]
WARNING:common:Trying to call the empty_gpu_cache for device: cpu, which is not in list [cuda, xpu]
WARNING:common:Trying to call the empty_gpu_cache for device: cpu, which is not in list [cuda, xpu]
E0801 03:36:27.017574 13946 torch/_dynamo/utils.py:1584] RMSE (res-fp64): 0.00157, (ref-fp64): 0.00046 and shape=torch.Size([1, 2]). res.dtype: torch.bfloat16, multiplier: 3.000000, tol: 0.001000
E0801 03:36:27.017788 13946 torch/_dynamo/utils.py:1476] Accuracy failed for key name logits
fail_accuracy
WARNING:common:Trying to call the empty_gpu_cache for device: cpu, which is not in list [cuda, xpu]
dev,name,batch_size,accuracy,calls_captured,unique_graphs,graph_breaks,unique_graph_breaks,autograd_captures,autograd_compiles,cudagraph_skips
cpu,LayoutLMForSequenceClassification,1,fail_accuracy,396,3,2,1,0,0,2
```
The last good commit:
ef8d118c677572bb14a39e9ee752acb375d2d558
```
/workspace/pytorch# bash inductor_single_run.sh single inference accuracy huggingface LayoutLMForSequenceClassification amp
Testing with freezing on.
single-thread testing....
/opt/conda/lib/python3.8/site-packages/huggingface_hub/file_download.py:1150: FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
  warnings.warn(
loading model: 0it [00:01, ?it/s]
cpu  eval  LayoutLMForSequenceClassification
WARNING:common:Trying to call the empty_gpu_cache for device: cpu, which is not in list [cuda, xpu]
WARNING:common:Trying to call the empty_gpu_cache for device: cpu, which is not in list [cuda, xpu]
WARNING:common:Trying to call the empty_gpu_cache for device: cpu, which is not in list [cuda, xpu]
pass
WARNING:common:Trying to call the empty_gpu_cache for device: cpu, which is not in list [cuda, xpu]
dev,name,batch_size,accuracy,calls_captured,unique_graphs,graph_breaks,unique_graph_breaks,autograd_captures,autograd_compiles,cudagraph_skips
cpu,LayoutLMForSequenceClassification,1,pass,396,3,2,1,0,0,2
```
Seems https://github.com/pytorch/pytorch/commit/8458980bbf78714a0fbe703785c100cad523fade#diff-004303ad6116d64ab2a8356469ccb11b32d8caca702e1ad65cc0538600a76d2dR508 needs to be modified to `and name in self._config["tolerance"]["higher_inference_cpu"]` 

### Versions

</table><p>SW info</p><table border="1" class="dataframe table">
  <thead>
    <tr style="text-align: right;">
      <th>name</th>
      <th>target_branch</th>
      <th>target_commit</th>
      <th>refer_branch</th>
      <th>refer_commit</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>torchbench</td>
      <td>main</td>
      <td>23512dbe</td>
      <td>main</td>
      <td>23512dbe</td>
    </tr>
    <tr>
      <td>torch</td>
      <td>main</td>
      <td>500aea8d5033fd3540c6ed325dd80e7e1420b0f3</td>
      <td>main</td>
      <td>1d1d074072ecb0aa6ca95e3f43221d2275e16d74</td>
    </tr>
    <tr>
      <td>torchvision</td>
      <td>main</td>
      <td>0.19.0a0+d23a6e1</td>
      <td>main</td>
      <td>0.19.0a0+d23a6e1</td>
    </tr>
    <tr>
      <td>torchtext</td>
      <td>main</td>
      <td>0.16.0a0+b0ebddc</td>
      <td>main</td>
      <td>0.16.0a0+b0ebddc</td>
    </tr>
    <tr>
      <td>torchaudio</td>
      <td>main</td>
      <td>2.4.0a0+69b2a0a</td>
      <td>main</td>
      <td>2.4.0a0+69b2a0a</td>
    </tr>
    <tr>
      <td>torchdata</td>
      <td>main</td>
      <td>0.7.1a0+0790338</td>
      <td>main</td>
      <td>0.7.1a0+0790338</td>
    </tr>
    <tr>
      <td>dynamo_benchmarks</td>
      <td>main</td>
      <td>nightly</td>
      <td>main</td>
      <td>nightly</td>
    </tr>
  </tbody>
</table>

</table>

Repro:
[inductor_single_run.sh](https://github.com/chuanqi129/inductor-tools/blob/main/scripts/modelbench/inductor_single_run.sh)
bash inductor_single_run.sh single inference accuracy huggingface LayoutLMForSequenceClassification amp
Suspected guilty commit: https://github.com/pytorch/pytorch/commit/8458980bbf78714a0fbe703785c100cad523fade
[huggingface-LayoutLMForSequenceClassification-inference-amp-static-default-single-accuracy-crash_guilty_commit.log](https://github.com/user-attachments/files/16450568/huggingface-LayoutLMForSequenceClassification-inference-amp-static-default-single-accuracy-crash_guilty_commit.log)
cc @ezyang @chauhang @penguinwu @WeizhuoZhang-intel @chuanqi129

### PR DESCRIPTION
**Pull request by zxd1997066**
Fixes #132380, adjust torchbench and huggingface skip models list, then we can remove `--no-skip` when running benchmarks on 3 suites.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang @amjames @rec

### CODE REVIEW

**Review by chuanqi129 — APPROVED** [review]

**Comment by chuanqi129** [comment]
Suggest to rename the title to "**Fix dynamo benchmark skip logic for cpu device**"

**Comment by chunyuan-w** [comment]
Hi @jansel @desertfire could you please help review this PR?

**Review by jansel — APPROVED** [review]
